server.js

```diff
@@ -41,16 +41,30 @@ const checkCredentials = async () => {
         throw new Error(error);
     }
 };
+
+
+const checkCampaign = async () => {
+    try {
+        await client.campaigns.get("Reward Promotion");
+    } catch (error) {
+        if (error.code === 404) {
+            const msg = "The 'Reward Promotion' campaign not found. This campaign is required for 'tiered-cart-promotion' to work properly. Please create a 'Reward Promotion' campaign first, you can check the details by visiting `https://docs.voucherify.io/reference/create-campaign`";
```

Comment:
Just wondering, can you give a hint about the `Reward Promotion` configuration? Maybe we can put an example `curl` that can create this promotion as a comment here so developers facing this issue can create it quickly?